src/models/community_business.ts

```diff
@@ -225,6 +234,14 @@ export const CommunityBusinesses: CommunityBusinessCollection = {
       .where(preProcessCb(client)(o));
   },
 
+  async fromOrganisation (client: Knex, o: Organisation) {
```

Comment:
This method is not needed as I've added organisation_id to the `CommunityBusinesses` model.